src/main/src/main/java/org/geoserver/security/GeoServerSecurityManager.java

```diff
@@ -2625,13 +2626,15 @@ boolean migrateFrom22(boolean migratedFrom21) throws Exception{
             
         // set redirect url after successful logout
         if (migratedFrom21== false)
```

Comment:
@bencaradocdavies Personally, I dislike "==false" instead of using "!". Even worse is "==true" ;)